pkg/microservice/aslan/core/common/repository/mongodb/image_tags.go

```diff
@@ -0,0 +1,110 @@
+package mongodb
+
+import (
+	"context"
+	"errors"
+
+	"go.mongodb.org/mongo-driver/bson"
+	"go.mongodb.org/mongo-driver/mongo"
+
+	"github.com/koderover/zadig/pkg/microservice/aslan/config"
+	"github.com/koderover/zadig/pkg/microservice/aslan/core/common/repository/models"
+	mongotool "github.com/koderover/zadig/pkg/tool/mongo"
+)
+
+type ImageTagsColl struct {
+	*mongo.Collection
+
+	coll string
+}
+
```

Comment:
consider create index for this table? Check EndureIndex method in other db query files